hd/lang/lexicon.txt

```diff
@@ -18548,5 +18572,6 @@ en: very short display
 fr: affichage très court
 
     remove %s column
+co: fà un cliccu nant’à st’intestatura per caccià sta o ste culonna(e) « %s »
 en: click on this header to remove the column(s) “%s”
-fr: cliquer sur cette en-tête pour supprimer la ou les colonne(s) « %s »
\ No newline at end of file
+fr: cliquer sur cette en-tête pour supprimer la ou les colonne(s) « %s »
```

Comment:
Pourquoi cette retouche ?

Reply:
Thanks for your review.
I just realized that I have suppressed the last `CR/LF` control character at the end of this line... but it wasn't on purpose! Shame on me ;-)
I'm going to undo this change.

Reply:
Please ignore my previous comment.
Frankly, I don't understand why this (last) string appears as being modified. I can't see the difference and I don't remember having the idea to modify it!

Reply:
I expected to reverse the change on the last line but my latest commit was unsuccessful.

Your help will be highly appreciated!

Reply:
Oh. Sorry, I didn't saw it was the last line and though it was a removal of unbreakables spaces between quotes.